test/visibility/import/rename/same_name_in_mod.good

```diff
@@ -1 +1,2 @@
-0
+same_name_in_mod.chpl:7: error: symbol F is multiply defined
+same_name_in_mod.chpl:1: note: also defined here
```

Comment:
As noted on one of the previous error messages, having something refer to line 5 would help the user understand what was going on here better.  And as mentioned there, that woldn't need to be addressed in this PR.

Reply:
https://github.com/Cray/chapel-private/issues/3166 will track the future work here